### DISCUSSION TITLE
Video don't resume when focus tab again in iPhone

### DISCUSSION THREAD

**Discussion by elrumordelaluz** [vimeo/player.js#1002] [Bug reports]
### Expected Behavior
When set as `loop: true`, `autoplay: true` and `autopause: false`, we expect the video continue when leave the tab or at least resume when the tab is focused again.

### Actual Behavior
The `play` event is fired but the player still paused.

### Steps to Reproduce

1. Goto https://player.vimeo.com/api/demo
1. Uncheck the `autopause` flag
1. Check the `loop` one
1. Set Mute

Then play the video, change tab for a second and get back to the tab with the _Player_

Here is a _screencast_:

https://user-images.githubusercontent.com/784056/167831579-7da7d655-7207-4bb0-8254-fc3da098e972.mp4

Looking the logs, seems that for some reason the video is `paused` when switching tab (and it is ok for me), an seems that the event `play` is fired, but the video it isn't.

**Comment by rkrishnan8594** [comment]
Thanks for filing this issue. Do you only observe this issue on iOS or have you been able to reproduce on other platforms?

**Comment by elrumordelaluz** [comment]
> have you been able to reproduce on other platforms?

I am able to reproduce only on this phone right now.

In desktop seems to work as expected, since don't pause when switching tab and continues playing.

**Comment by sssalv** [comment]
Any news on this bug?

**Comment by siarheipashkevich** [comment]
Any news?

**Comment by bdougherty** [comment]
I'm not able to reproduce this on iOS 17.4.1. Is anyone still able to reproduce?
